data_platform_helpers/tests_helpers/__init__.py

```diff
@@ -1,10 +1,9 @@
 import logging
+from importlib.util import find_spec
 
-try:
-    import pytest_operator
-except ImportError as e:
```

Comment:
Linting complains over the unused import, using find_spec to check if pytest_operator is installed